text/0009-metric-handles.md

```diff
@@ -1,74 +1,36 @@
 # Metric Handle API specification
 
-**Status:** `proposed`
+**Status:** `accepted`
 
-Specify the behavior of the Metrics API `Handle` type, for efficient repeated-use of metric instruments.
+Specify the behavior of the Metrics API "Handle" type, for efficient repeated-use of metric instruments.
 
 ## Motivation
 
-The specification currently names this concept `TimeSeries`, the object returned by `GetOrCreateTimeseries`, which supports binding a metric to a pre-defined set of labels for repeated use.  This proposal renames these `Handle` and `GetHandle`, respectively, and adds further detail to the API specification for handles.  
+The specification currently names this concept "TimeSeries", the type returned by `GetOrCreateTimeseries`, which supports binding a metric to a pre-defined set of labels for repeated use.  This proposal renames these "Handle" and `GetHandle`, respectively, and adds further detail to the API specification for handles.  
```

Comment:
I think you are using `acquireHandle` in Go. Is that a better name?

Reply:
True. If you'd like me to continue updating this OTEP inside of this PR, I will. The specification says "GetHandle" currently.

During review of the current specification, Yuri requested that we renamed handles to "Bound instruments" and "GetHandle" to "Bind". I was hoping to put this proposal in a future OTEP, combined with a specification for a "Release" method to free a binding.

Reply:
ACK.